roles/build_openstack_packages/tasks/install_dlrn.yml

```diff
@@ -14,33 +14,14 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Install epel-release repo
+- name: Install Mock
```

Comment:
we may want to install from EPEL if we don't have dlrn? Maybe a `block/except` to try installing from existing repositories, and fall back on EPEL in case it can't find it?